src/idl2json/Cargo.toml

```diff
@@ -12,20 +12,19 @@ readme = "../../README.md"
 
```

Comment:
This is just formatting by `cargo sort`.  A bit surprised; presumably the Cargo.toml file formatting is not checked in CI.  If so, this is a small TODO.